getl/blocks/write/entrypoint.py

```diff
@@ -147,10 +150,14 @@ def batch_delta(conf: BlockConfig) -> DataFrame:
     """
     path = conf.get("Path")
     mode = conf.get("Mode")
+    columns = conf.get("PartitionBy.Columns", None)
     dataframe = conf.history.get(conf.input)
     batch = BatchDelta(dataframe, conf.spark)
     htable = None
 
+    if columns:
+        columns = columns.replace(" ", "").split(",")
```

Comment:
Why not require columns to already be a list of strings? Like `write::batch_postgres_upsert` Columns?